api/preprints/views.py

```diff
@@ -164,7 +164,29 @@ def get_annotated_queryset_with_metrics(self, queryset, metric_class, metric_nam
         )
 
 
-class ListCreatePreprintVersion(PreprintList):
+class PreprintVersionsList(PreprintMetricsViewMixin, JSONAPIBaseView, generics.ListCreateAPIView, PreprintFilterMixin):
+    # These permissions are not checked for the list of preprints, permissions handled by the query
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        ContributorOrPublic,
+    )
```

Comment:
Reminder for myself: it seems neither the `create` nor the `list` checks the permission ... if the comment is correct ... need to double check ... I think these are still checked but just not strict enough, which finer-grained access control are done in `get_queryset()`.